code/clue.js

```diff
@@ -1,59 +1,304 @@
+  lastseen: "Miss Scarlet",
+  favespot: "Mrs Grey"
+};
+
+const billiardRoom = {
+  name: "Billiard Room",
+  color:"#d78989",
+  image: "assets/billiardroom.jpg",
+  lastseen: "Colonel Mustard",
+  favespot: "Professor Plum"
+};
+
+const livingRoom = {
+  name: "Living room",
+  color:"#e59e34",
+  image: "assets/livingroom.jpg", 
+  lastseen: "Mrs Peacock",
+  favespot: "Colonel Mustard"
+};
+
+// GROUPED SUSPECTS, WEAPONS AND ROOM OBJECTS IN ARRAYS
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+  mrsGrey, 
+];
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  poison,
+  axe,
+  pokerstick,
+  pistol
+];
+
+const rooms = [
+  diningRoom,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  livingRoom,
+];
+
+// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
+// With a killer, a weapon and a room.
+// The values will be set later.
 
-const rooms = []
+const mystery = {
+  killer: {},
+  weapon: {},
+  room: {}
+};
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
 // YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
-}
+};
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+//----- Killer card -----//
+//Function is called when user clicks on card. The animation is run for 1.5 seconds, which delays the showPickKiller function from being called for that duration. Once over it will then call the showPickKiller function and the code inside of it e.g. show the card-details for that killer. The animation will be done by adding the animated-loader styling to the loader1 id.
+const pickKillerLoader = () => {
+  document.getElementById("loader1").classList.add("animated-loader");
+  setTimeout(showPickKiller, 2000);
+};
+
+//On click the function pickKillerLoader will be called by using the addEventListner()
+document.getElementById("killerCard").addEventListener("click", pickKillerLoader); 
+
+// Function invoked after the pickKillerLoader has run
+const showPickKiller = () => {
+  mystery.killer = randomSelector(suspects);// Will randomly select a killer from the suspects + add that to the mystery object.
+  document.getElementById("loader1").style.display = "none";//Will hide the loader image
+
+  document.getElementById('killerCard').removeEventListener('click', pickKillerLoader); //Will stop the loader showing next time you click the card
+
+  // Will change the background color of the card, the first/last name, the image and descrption for the selected killer. 
+  document.getElementById("killerCard").style.background = mystery.killer.color;
+  document.getElementById("killerName").innerHTML = `${mystery.killer.name}`;
+  document.getElementById("killerImage").src = mystery.killer.image;
```

Comment:
**_(To the code coach who'll be reviewing my review; does it sounds like that would make sense?)_** - Yes Axel, that do make sense, good assumption :)

**_(And, another nit-picking note (sorry!): for consistency, try to avoid mixing ' and ")_** Agree as well!